[esx]/esx_menu_default/client/main.lua

```diff
@@ -1,93 +1,116 @@
-local GUI, MenuType = {}, 'default'
+local GUI, MenuType = {}, "default"
```

Comment:
very optimised

Reply:
You should take a look on commits. As you can see, there is a commit with reformatting.

Reply:
But why in a different stlye than the rest of ESX Legacy, doesnt make sense